api/src/iot/import_utils.py

```diff
@@ -713,6 +721,15 @@ def import_xlsx(workbook, action_logger=lambda x: x):
     parser = parse_bulk_xlsx if "Uw gegevens" in workbook else parse_iprox_xlsx
     sensors = list(parser(workbook))
 
+    sensor_counter = Counter(sensor.reference for sensor in sensors)
+    duplicate_references = [
+        (reference, count)
+        for reference, count in sensor_counter.most_common()
```

Comment:
Je zou hier eventueel een argument mee kunnen geven aan `most_common()` zodat je bijvoorbeeld maximaal 5 results krijgt.

Reply:
Good idee! Ik bedenk me nu trouwens dat dit voor alle fouten geldt. Als een bestand 100 fouten bevat worden 100 berichten gestuurd. Misschien is het dan beter om hier toch alles terug te geven en te limiteren bij het tonen van de fouten.